test/e2e/framework/pod/wait.go

```diff
@@ -51,7 +51,7 @@ const (
 	podScheduledBeforeTimeout = podListTimeout + (20 * time.Second)
 
 	// podStartTimeout is how long to wait for the pod to be started.
-	podStartTimeout = 5 * time.Minute
+	podStartTimeout = 8 * time.Minute
```

Comment:
my only concern is that this changes the timeout for all tests. Is there a way we can have a separate timeout for windows?

Reply:
it is possible to do it, I think. But maybe just leave it 5 mins for now and see how flaky the tests will be and adjust later.